pkg/scheduler/partition.go

```diff
@@ -1314,6 +1317,8 @@ func (pc *PartitionContext) removeAllocation(release *si.AllocationRelease) ([]*
 				zap.String("appID", appID),
 				zap.String("allocationId", uuid),
 				zap.Error(err))
+		} else {
+			metrics.GetQueueMetrics(queue.QueuePath).IncReleasedContainer()
```

Comment:
I think this should be a GetQueuePath() call and not a direct access to the QueuePath